addons/mail/static/src/components/file_uploader/file_uploader.js

```diff
@@ -151,6 +152,7 @@ export class FileUploader extends Component {
         }
         const attachment = this.messaging.models['Attachment'].insert({
             composer: composer && replace(composer),
+            messageCompositions: composer && composer.messageComposition && link(composer.messageComposition),
```

Comment:
same as before, should delete `composer:`

There should be a crash, but it doesn't happen. I think @oomsveta applied a fix recently, so if you rebase you should have proper crash.